camel/configs.py

```diff
@@ -74,3 +74,71 @@ class ChatGPTConfig:
     frequency_penalty: float = 0.0
     logit_bias: Dict = field(default_factory=dict)
     user: str = ""
+
+
+@dataclass
+class FuncConfig:
```

Comment:
This is an extraordinary code duplication. Not good to copy-paste a huge chunk of code and change one line. Please rework with either composition or inheritance.